### PR TITLE
[docs]: Tweaks and fixes in dark theme

### PR DESCRIPTION
**Pull request by Caladan08**
### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing guide](https://github.com/svelteuidev/svelteui/blob/main/CONTRIBUTING.md)
- [x] Prefix your PR title with `[@svelteui/core]`, `[@svelteui/actions]`, `[@svelteui/motion]`, `[@svelteui/core]`, `[core]`, or `[docs]`.
- [x] Include a description of the changes made in the PR description and in the commit messages.
- [ ] Include screenshots/videos of the changes made.
- [ ] Verify that the linter and tests are passing, with `yarn lint` and `yarn test` or just run `yarn prepush`.

There were quite a few bugs with components and this fixes it

### CODE REVIEW

File: apps/docs/src/lib/theme/styles/layout.css [hunk 1]
```diff
@@ -344,6 +344,19 @@ body {
...
+.dark-theme .sibling,
+.dark-theme .guides,
+.dark-theme .packages,
+.dark-theme .next_steps {
+  background-color: #191b1e !important;
+  color: white !important
+}
+
+.dark-theme .sidebar ul li a.main-link.active {
+  background-color: #000000;
```
Comment (BeeMargarida) [inline apps/docs/src/lib/theme/styles/layout.css:361]: I would change to the code below, to make it similar to the sections below

```
background-color: rgba(24, 100, 171, 0.45);
color: var(--svelteui-colors-white);
```

File: apps/docs/src/lib/theme/ui/Sibling/Sibling.styles.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { createStyles } from '@svelteuidev/core';
 export default createStyles((theme, _params, getRef) => {
 	return {
 		root: {
-			[`${theme.dark} &`]: {
+			[`${theme.dark}`]: {
```
Comment (BeeMargarida) [inline apps/docs/src/lib/theme/ui/Sibling/Sibling.styles.js:6]: you can just switch to `darkMode: {`. The same below in the text style

**Review by BeeMargarida — COMMENTED** [review]
Thank you for these fixes! Just left some small comments

**Comment by Caladan08** [comment]
There's another problem with the dark theme when you visit the "Dark Theme" page in the Theming section where when you visit it, even if you have dark theme enabled, it automatically switches to light mode. How should I go about fixing it?

https://user-images.githubusercontent.com/94300963/233678875-0cd1ee56-e29b-4a9a-a9ac-6eb999d92704.mov

**Comment by BeeMargarida** [comment]
> There's another problem with the dark theme when you visit the "Dark Theme" page in the Theming section where when you visit it, even if you have dark theme enabled, it automatically switches to light mode. How should I go about fixing it?

I'm fixing that right now, it's a outdated decision in the dark theme demo. Thank you for pointing it out!
One thing, if you want to tackle it in this PR, is fixing the navigation in the search modal: when navigating from there to a component page and then to a composable page, it breaks because it builds a wrong URL, like `/core/composable/use-clipboard` instead of `composable/use-clipboard`

**Comment by Caladan08** [comment]
It seems to be working for me. All the links in `data.ts` look good as well. Which two links did you visit that broke the code?

**Comment by BeeMargarida** [comment]
> It seems to be working for me. All the links in `data.ts` look good as well. Which two links did you visit that broke the code?

An example in video, since it's easier to explain: 

https://user-images.githubusercontent.com/25725586/233776312-cbef0801-d1cc-476f-b86b-f1b3233727e7.mp4

**Comment by Caladan08** [comment]
This is how it looks on my side:

https://user-images.githubusercontent.com/94300963/233797150-1ae4d6f7-6316-4142-a573-5a72de7ff481.mov

**Comment by BeeMargarida** [comment]
Nvm, had to clean my local storage

**Review by BeeMargarida — APPROVED** [review]

**Comment by BeeMargarida** [comment]
Thank you for this contribution!! :raised_hands:

**Pull request merged**